tortoise/verifying_tortoise.go

```diff
@@ -84,13 +67,11 @@ func (t *turtle) init(genesisLayer *types.Layer) {
 	t.BlockOpinionsByLayer[genesisLayer.Index()] = make(map[types.BlockID]Opinion)
 	for _, blk := range genesisLayer.Blocks() {
 		id := blk.ID()
-		t.BlockOpinionsByLayer[genesisLayer.Index()][blk.ID()] = Opinion{
-			BlocksOpinion: make(map[types.BlockID]vec),
-		}
-		t.GoodBlocksIndex[id] = struct{}{}
+		t.BlockOpinionsByLayer[genesisLayer.Index()][id] = Opinion{}
```

Comment:
don't you still need to initialize the `Opinion` map?

Reply:
Opinion is a map now

Reply:
exactly. don't you need to initialize it with `make`?

Reply:
but Opinion{} and make(Opinion) will do the same thing for map, unless you want to specify length